capistrano-deploy-all.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |spec|
   spec.platform    = Gem::Platform::RUBY
   spec.name        = "capistrano-deploy-all"
-  spec.version     = "0.1.1"
+  spec.version     = "0.1.4"
```

Comment:
I think we should go for `0.2.0` version following the semantic versioning because this is a new feature without a breaking change. https://semver.org/